tests/production/Common.js

```diff
@@ -0,0 +1,141 @@
+/**
+Common data and code shared among the production configuration tests.
+
+Copyright 2019 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("%gpii-universal");
+fluid.registerNamespace("gpii.tests.productionConfigTesting");
+
+gpii.tests.productionConfigTesting.config = {
+    configName: "gpii.tests.untrusted.production.config",
+    configPath: "%gpii-universal/tests/configs"
+};
+
+// Grade for "disruptions" that are also proper sequence grades.  Use the
+// standard server sequence
+fluid.defaults("gpii.test.disruption.sequenceGrade", {
```

Comment:
Same as above that this same grade has been defined.

Reply:
See below.